sdk/documentintelligence/azure-ai-documentintelligence/CHANGELOG.md

```diff
@@ -2,4 +2,129 @@
 
 ## 1.0.0b1 (2023-11-17)
 
-- This is the first preview of the `azure-ai-documentIntelligence` package, targeting API version 2023-10-31-preview of the Document Intelligence service.
+This is the first preview of the `azure-ai-documentintelligence` package, targeting API version `2023-10-31-preview` of the Document Intelligence service(formerly known as Form Recognizer).
```

Comment:
To me, this content looks more appropriate for a migration guide than a changelog entry. We can always link to the migration guide from the changelog.